app/src/components/user_schema_table.rs

```diff
@@ -18,9 +28,20 @@ pub struct GetUserAttributesSchema;
 use get_user_attributes_schema::ResponseData;
 
 pub type Attribute = get_user_attributes_schema::GetUserAttributesSchemaSchemaUserSchemaAttributes;
-pub type AttributeType = get_user_attributes_schema::AttributeType;
 
-pub struct UserAttributesTable {
+convert_attribute_type!(get_user_attributes_schema::AttributeType);
+
+fn sort_with_hardcoded_first(a: &Attribute, b: &Attribute) -> Ordering {
+    if a.is_hardcoded && !b.is_hardcoded {
```

Comment:
This code will probably change, but I think you can replace the whole body with `(!a.is_hardcoded, &a.name).cmp(&(!b.is_hardcoded, &b.name))`

And then you can even inline it if you want.